.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Delete all of your `.idea` files and ensure gitignore ignores them please. They're cluttering up this PR.

Reply:
done